packages/components/src/layouts/Navigation/Header/HeaderView.tsx

```
@@ -11,11 +12,13 @@ import platformEnv from '@onekeyhq/shared/src/platformEnv';
 import { EPageType, usePageType } from '../../../hocs';
 import { Stack, XStack } from '../../../primitives';
 import { DesktopDragZoneBox } from '../../DesktopDragZoneBox';
+import { rootNavigationRef } from '../Navigator';
```

ESLint

[error] Dependency cycle via ./RootModalNavigator:1=>../GlobalScreenOptions:5=>./Header:8

(import/no-cycle)